Header header

float64 gateway_double
string  gateway_string
string  gateway_key
time    gateway_time
